src/lib/db/feature-toggle-store.ts

```diff
@@ -289,8 +290,16 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
             return this.rowToFeature(row[0]);
         } catch (err) {
             this.logger.error('Could not insert feature, error: ', err);
+            if (
```

Comment:
when we create feature and SQL fails we should not fail silently and return undefined. We want to get instant feedback from code that something was wrong where the failure happened